src/other/tables.jl

```diff
@@ -11,27 +11,32 @@ Tables.materializer(df::AbstractDataFrame) = DataFrame
 
 getvector(x::AbstractVector) = x
 getvector(x) = collect(x)
-fromcolumns(x) = DataFrame(AbstractVector[getvector(c) for c in Tables.eachcolumn(x)], Index(collect(Symbol, propertynames(x))))
+fromcolumns(x; copycolumns::Bool=true) =
```

Comment:
Do we want to allow `copycolumns` kwarg here?

Reply:
@quinnj?

Reply:
Yeah, I think having `copycolumns` would be good here, in case they're going from a columnar to DataFrame and want to modify w/o affecting the original source.